### PR TITLE
ADD: restart-services role & github actions molecule schedule

### PR DESCRIPTION
**Pull request by stefa2k**

### CODE REVIEW

**Review by gbayasgalan — APPROVED** [review]
lg

File: launcher/src/backend/NodeConnection.js [hunk 1]
```diff
@@ -857,13 +860,42 @@ export class NodeConnection {
...
+      let after = await this.getTimeStamp()
+      return parseInt(after) - parseInt(before)
     } catch (err) {
       log.error("Error occurred running stereum updates:\n", err)
     }
   }
 
+  async getTimeStamp(){
+    try {
+      let val = await this.sshService.exec('date +%s')
```
Comment (stefa2k) [inline launcher/src/backend/NodeConnection.js:875]: You could just take the time diff off of the local desktop. Would save a round to ssh command.

Comment (NeoPlays) [reply launcher/src/backend/NodeConnection.js:875]: I was a bit in a hurry doing this 😅
Improving it as soon as i'm back home 👍🏼

**Comment by NeoPlays** [comment]
Services restart only after update when `autoupdate=true` is included in the config.
This is not yet implemented so i decided to take the timestamp after service updates so that updated services also get rebooted.

**Review by NeoPlays — APPROVED** [review]

**Pull request merged**